pkg/kn/commands/types.go

```diff
@@ -31,12 +31,17 @@ var KubeCfgFile string
 type KnParams struct {
 	Output         io.Writer
 	ServingFactory func() (serving.ServingV1alpha1Interface, error)
+	// CurrentNamespace is the namespace set in the current context.
```

Comment:
I think it's clear what CurrentNamespace is. So please either remove the comment (recommended) or comment on the other parameters, too.